teams/test-tools.toml

```diff
@@ -0,0 +1,22 @@
+name = "test tools"
+subteam-of = "devtools"
+
+[people]
+leads = ["calebcartwright"]
+members = [
+    "calebcartwright",
+    "epage",
+    "muscraft",
+    "thomcc",
+    "weihanglo",
+]
+
+[[github]]
+orgs = ["rust-lang"]
+
+[website]
+name = "Test tools team"
+description = "Defining strategy and associated tooling for supporting automated testing activities in the development workflow"
+
+[[zulip-groups]]
+name = "T-test-tools"
```

Comment:
I'll work with the infra team to make sure this is created once the team name is settled

Reply:
FYI: This will be created automatically.